src/udev/udev-builtin-path_id.c

```diff
@@ -543,19 +543,51 @@ static sd_device *handle_ap(sd_device *parent, char **path) {
         return skip_subsystem(parent, "ap");
 }
 
+static int find_real_nvme_parent(sd_device *dev, sd_device **ret) {
+        _cleanup_(sd_device_unrefp) sd_device *nvme = NULL;
+        const char *sysname, *end;
+        int r;
+
+        /* If the device is belonging to "nvme-subsystem" (instead of "nvme", do not cofuse), which
+         * happens when multipath is enabled, then the syspath is something like the following:
+         *   /sys/devices/virtual/nvme-subsystem/nvme-subsys0/nvme0n1
+         * Hence, we need to find the 'real parent' in "nvme" subsystem, e.g,
+         *   /sys/devices/pci0000:00/0000:00:1c.4/0000:3c:00.0/nvme/nvme0 */
```

Comment:
`If the device belongs to "nvme-subsystem" (not to be confused with "nvme"), which happens when multipath is enabled, then the syspath is something like the following:`